src/main/java/Duke.java

```diff
@@ -1,10 +1,129 @@
+import java.util.Arrays;
+import java.util.Scanner;
 public class Duke {
+
+    public static Task[] addTask(Task[] tasks, Task task){
+        tasks = Arrays.copyOf(tasks, tasks.length + 1);
+        tasks[tasks.length - 1] = task;
+        return tasks;
+    }
+
+    public static String joinString(String[] strings, int idx){
+        String joinedString = "";
+        for (int i = idx; i < strings.length; i++){
+            if (strings[i] == null){
+                break;
+            } else {
+                joinedString += strings[i] + " ";
+            }
+        }
+        return joinedString.trim();
+    }
+    public static void welcome(){
```

Comment:
maybe `welcomeMessage` is a better name? Since it does not seem to have any other function